### PR TITLE
Docs/markdown for website

### PR DESCRIPTION
**Pull request by fSosaRey**
Standardize filenames and reorganize `docs/src/library` into multiple files

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -32,25 +31,43 @@ DocMeta.setdocmeta!(
...
-            "Real hardware" =>"tutorials/run_circuit_anyon.md"
-        ],
-        "Advanced Examples" => [
-            "Asynchronous Jobs"=>"tutorials/advanced/async_jobs.md"
+
+formattersAndDirs=[
+    (Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "false"), "build", "HTMLWriter"),
+]
+
+for (formatter,build_dir, name) in formattersAndDirs
```
Comment (plerouxanyon) [inline docs/make.jl:39]: Do we need this loop if we only have one possible option?

**Review by plerouxanyon — APPROVED** [review]

File: src/core/quantum_gate.jl [hunk 2]
```diff
@@ -1474,13 +1474,13 @@ phase_shift(phi,T::Type{<:Complex}=ComplexF64) = DiagonalOperator(T[1.,exp(im*ph
     universal(theta, phi, lambda)
 
 Return the `Operator` which performs a rotation about the angles `theta`, `phi`, and `lambda`.
-See: https://qiskit.org/textbook/ch-states/single-qubit-gates.html#generalU
+See: Theorem 4.1 in [Quantum Computation and Quantum Information by Nielsen and Chuang](http://mjg.global.s3.amazonaws.com/static/projects/refs/Quantum_Computation_and_Quantum_Information.pdf).
```
Comment (plerouxanyon) [inline src/core/quantum_gate.jl:1477]: I think we should rather link to the Wikipedia page

File: src/core/quantum_gate.jl [hunk 3]
```diff
@@ -1863,7 +1863,7 @@ get_gate_parameters(gate::PhaseShift)=Dict("lambda" =>gate.phi)
     universal(target, theta, phi, lambda)
 
 Return a gate which rotates the `target` qubit given the angles `theta`, `phi`, and `lambda`.
-See: https://qiskit.org/textbook/ch-states/single-qubit-gates.html#generalU
+See: Theorem 4.1 in [Quantum Computation and Quantum Information by Nielsen and Chuang](http://mjg.global.s3.amazonaws.com/static/projects/refs/Quantum_Computation_and_Quantum_Information.pdf).
```
Comment (plerouxanyon) [inline src/core/quantum_gate.jl:1866]: Same as above

**Review by plerouxanyon — CHANGES_REQUESTED** [review]

File: src/core/quantum_gate.jl (hunk 2 above)
Comment (fSosaRey) [reply src/core/quantum_gate.jl:1477]: do you mean [this](https://en.wikipedia.org/wiki/Quantum_logic_gate) page? Unlike the textbook, this page does not mention the U-gate specifically

File: src/core/quantum_gate.jl (hunk 3 above)
Comment (fSosaRey) [reply src/core/quantum_gate.jl:1866]: changed

File: src/core/quantum_gate.jl (hunk 2 above)
Comment (fSosaRey) [reply src/core/quantum_gate.jl:1477]: changed

**Review by plerouxanyon — APPROVED** [review]

**Pull request merged**